oneview_redfish_toolkit/api/capability.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from oneview_redfish_toolkit.api.redfish_json_validator \
+    import RedfishJsonValidator
+
+
+class Capability(RedfishJsonValidator):
+    """Creates a Capability Redfish dict
+
+        Populates self.redfish with some hardcoded Capability
+        values and with the response of OneView.
+    """
+
+    SCHEMA_NAME = 'CollectionCapabilities'
+    BASE_URI = '/redfish/v1/System'
+
+    def __init__(self, profile_template):
```

Comment:
I think is missing 'Args' documentation here.

Reply:
Added args documentation